client/jetpack-cloud/sections/partner-portal/license-product-card/style.scss

```diff
@@ -0,0 +1,72 @@
+.license-product-card {
+	width: 33%;
+}
+
+.license-product-card__inner {
+	margin: 0.5rem;
+	padding: 1.5rem;
+	border: 1px solid #ddd;
+	border-radius: 2px;
+	cursor: pointer;
+}
+
+.license-product-card.selected .license-product-card__inner {
+	border: 1px solid #fff;
+	background: #fff;
+	box-shadow: 0 0 12px #ddd;
+}
+
+.license-product-card__title {
+	font-size: 1.5rem;
+	margin-bottom: 1rem;
+	color: #444;
+	display: inline-block;
+}
+
+.license-product-card__radio {
+	border: 2px solid #444;
+	width: 16px;
+	height: 16px;
+	display: inline-block;
+	float: right;
+	border-radius: 10px; /* stylelint-disable-line */
+}
+
+.license-product-card__radio .gridicon {
+	color: #fff;
+	position: relative;
+	top: -4px;
+	left: 1px;
+	width: 14px;
+}
+
+.license-product-card.selected .license-product-card__radio {
+	border: 2px solid green;
+	background: green;
+}
+
+.issue-license-form p.license-product-card__text {
+	font-size: 0.75rem;
+	color: #777;
+	margin-bottom: 0;
+}
+
+.issue-license-form__top {
+	display: flex;
+	justify-content: space-between;
+}
+
+.issue-license-form__top > div {
+	align-self: flex-end;
+	margin-bottom: 0;
+}
+
+@media ( max-width: 960px ) {
```

Comment:
Some of the code uses the Gutenberg breakpoints, which has `960px` as an option:
```
@import '@wordpress/base-styles/_breakpoints.scss';
@import '@wordpress/base-styles/_mixins.scss';
```

Should we stick with how the rest does it or do we differ in some way where we don't want to follow them (just because they exist as a variable/mixin)?

Reply:
Ah! I stumbled upon https://wpcalypso.wordpress.com/devdocs/docs/coding-guidelines/css.md#media-queries
but I don't know if "Calypso green" is strictly following the Calypso blue code standards?

Reply:
[This](https://wpcalypso.wordpress.com/devdocs/docs/coding-guidelines/css.md#media-queries) literally does not work. I'm not sure why, but I went with the existing pattern in Calypso Green. It should be fine for now, though we could probably afford to do a spike on styling patterns at some point in the future.

Reply:
I got curious (wups 😅 ) and tried it out myself. I've got it working and pushed a couple of commits.
While I was at it, also increased the breaking from `960px` to `1280px` because the cards would look at bit chaotic until they collapsed in smaller devices:

**Before**
![Screenshot 2022-02-04 at 15 31 43](https://user-images.githubusercontent.com/3846700/152549596-beed929a-8348-47ea-927f-f5c41da2d3fe.png)

**After**
![Screenshot 2022-02-04 at 15 55 30](https://user-images.githubusercontent.com/3846700/152550396-67d2cf2d-8418-4aa0-b097-0f9ac8e2d0c8.png)

Reply:
Moving this to "Unresolved" again to ensure you see it @rcoll - feel free to close it again 😄